sdk/eventhubs/azure_messaging_eventhubs/examples/eventhubs_produce_events.rs

```diff
@@ -1,12 +1,12 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-/// This sample demonstrates how to send events to an Event Hub partition using the `ProducerClient`.
-///
-use core::f32;
 use azure_core::Uuid;
 use azure_identity::DefaultAzureCredential;
 use azure_messaging_eventhubs::{models::EventData, ProducerClient, SendEventOptions};
+/// This sample demonstrates how to send events to an Event Hub partition using the `ProducerClient`.
+///
+use core::f32;
```

Comment:
Nit: same here. I'll not repeat, but I would think a comment describing your example should be at the very top so it's plainly visible. Not sure if this was a rust-analyzer thing or something, but using `//!` might also help prevent that since rust-analyzer always treats those as "opening comments" for the module (as they are normally under `src/`).